yarn.lock

```diff
@@ -17447,7 +17447,7 @@ ejs@^3.1.10:
   dependencies:
     jake "^10.8.5"
 
-elastic-apm-node@^4.10.0, elastic-apm-node@^4.12.0:
+elastic-apm-node@^4.10.0:
```

Comment:
Quick checks will deduplicate the `yarn.lock` file... We'll need to re-run `/ci`